test/libyul/yulOptimizerTests/reasoningBasedSimplifier/addmod.yul

```diff
@@ -26,6 +26,9 @@
 //     let z := calldataload(64)
 //     let result := addmod(x, y, z)
 //     if 0 { }
-//     if and(and(lt(x, 1000), lt(y, 1000)), lt(z, 1000)) { if 1 { sstore(0, 9) } }
+//     if and(and(lt(x, 1000), lt(y, 1000)), lt(z, 1000))
+//     {
+//         if eq(result, mod(add(x, y), z)) { sstore(0, 9) }
+//     }
```

Comment:
Looks like this needs a rebase. These are SMT-related changes that I think were already fixed on `develop`.